template/build/dev-server.js

```diff
@@ -1,3 +1,5 @@
+if (!process.env.NODE_ENV) process.env.NODE_ENV = 'development'
```

Comment:
The sec server is also used for testing. Is it wise to override the 'testing' environment?